resources/assets/sass/app.scss

```diff
@@ -12,3 +12,4 @@
 @import "nav";
 @import "profile";
 @import "wysiwyg";
+@import "forum";
```

Comment:
Ah sorry, I missed this earlier. Can you place forum right underneath "footer" so they're ranked alphabetically?